enioka_scan/src/main/java/com/enioka/scanner/sdk/athesi/HHTScanner.java

```diff
@@ -50,8 +50,23 @@
     @Override
     protected void configureProvider() {
         broadcastIntentFilters.add("DATA_SCAN");
-        disableTrigger = newIntent(DataWedge.SOFTSCANTRIGGER, DataWedge.EXTRA_PARAMETERS, new String[]{DataWedge.DISABLE_TRIGGERBUTTON, DataWedge.STOP_SCANNING});
-        enableTrigger = newIntent(DataWedge.SOFTSCANTRIGGER, DataWedge.EXTRA_PARAMETERS, new String[]{DataWedge.ENABLE_TRIGGERBUTTON, DataWedge.START_SCANNING});
+        // FIXME: Too many params (stop/start scanning is unrelated to pause/resume), and trigger activation/deactivation seems to have no effect after testing.
```

Comment:
actually no. You want to be sure that scanning is resumed when enabling trigger, otherwise it may result in a very situation wwhere scanner is disabled but trigger enabled...

Reply:
In this case the `TRIGGERBUTTON` enabling/disabling seems to affect whether the physical button is recognized as a trigger or not (and after testing does not appear to work), whereas `SCANNING` start/stop acts as a press on the trigger, not as enabling/disabling the ability to scan at all, which seem to be what enabling/disabling `PLUGIN` appears to be doing

Reply:
Resolved: replaced by correct intent call to properly disable scanning